cs/ccxt/base/Throttler.cs

```diff
@@ -8,8 +8,10 @@ public class Throttler
 
     private dict config = new dict();
     private Queue<(Task, double)> queue = new Queue<(Task, double)>();
+    private readonly object queueLock = new object();
 
     private bool running = false;
+    private List<(long timestamp, double cost)> timestamps = new List<(long, double)>();
```

Comment:
@carlosmiei what about having ConcurrentList here instead of the regular list? or SlimList even?